docs/src/user/manual/landing-page.md

```diff
@@ -36,25 +36,36 @@ Ready to explore the health of the local river? Grab the collection kit, follow
 
     ![Blog](./img/landing-page-4.png)
 
-10. **EXPLORE THE MAP**: Click on the `EXPLORE THE MAP` card section, to explore the interactive map.
+10. **Download miniSASS App**: Click on the `Download miniSASS App` to, navigate to the download miniSASS mobile app page. After clicking on the `Download miniSASS App` button, the user will be redirected to the mobile app page.
+
+    **Mobile App:** If you are using a mobile device, you can access these functionalities through the app.
+
+    ![Mobile App](./img/landing-page-20.png)
+
+    1. **DOWNLOAD MINISASS APP HERE:** Click on the `DOWNLOAD MINISASS APP HERE` button, to download the miniSASS APK.
+
+    2. **How to Manually Install the MiniSASS App APK:** This guide provides step-by-step instructions on how to manually install the MiniSASS app on an Android device using its APK file. Users are reminded to exercise caution when installing apps from outside the Google Play Store, emphasising the importance of downloading APK files from trusted sources. The key steps include enabling installation from unknown sources in device settings, downloading the MiniSASS APK from a reputable website, locating the downloaded file in the device's storage, installing the app, and launching it. Additionally, users are advised to restore their security settings after installation and regularly check for updates manually since the app won't update automatically.
```

Comment:
```suggestion
    2. **How to Manually Install the MiniSASS App APK:** This guide provides step-by-step instructions on how to manually install the miniSASS app on an Android device using its APK file. Users are reminded to exercise caution when installing apps from outside the Google Play Store, emphasising the importance of downloading APK files from trusted sources. The key steps include enabling installation from unknown sources in device settings, downloading the miniSASS APK from a reputable website, locating the downloaded file in the device's storage, installing the app, and launching it. Additionally, users are advised to restore their security settings after installation and regularly check for updates manually since the app won't update automatically.
```